nyaml/nyaml2nxdl.py

```diff
@@ -526,7 +526,12 @@ def xml_handle_dim_from_dimension_dict(
             if not isinstance(llist_ind_value, list):
                 if llist_ind_value.startswith("(") and llist_ind_value.endswith(")"):
                     llist_ind_value = llist_ind_value[1:-1].split(",")
-                    llist_ind_value = tuple([x.strip() for x in llist_ind_value])
+                    llist_ind_value = (
```

Comment:
So, the provided "llist_ind_value" has always two entries.
If the last entry is empty, then remove the last entry? Which is fine if only a single dim entry is present.

Not sure if I understood this correctly, but sounds fine fine for me. Not tested it myself explicitly.

Reply:
You are right. If you go to the issue #37, you see because of empty element (last element) of the list we get an XML element `dim` which does not have a `value`. Which is wrong. 
In this PR, I cleaned the `empty` element from the list and therefore it does not create any `value` less XML element `dim`.

Reply:
yea okay. Just for me for claricitcation, as I did not work with this before. Idea is simple and works fine this way. looks fine to me.